src/stellar/StellarTools.ts

```diff
@@ -81,11 +87,33 @@ export class StellarTools implements EdgeCurrencyTools {
     return { stellarKey: keypair.secret() }
   }
 
-  async importPrivateKey(privateKey: string): Promise<{ stellarKey: string }> {
-    privateKey.replace(/ /g, '')
-    stellarApi.Keypair.fromSecret(privateKey)
-    if (privateKey.length !== 56) throw new Error('Private key wrong length')
-    return await Promise.resolve({ stellarKey: privateKey })
+  async importPrivateKey(userInput: string): Promise<StellarPrivateKeys> {
+    if (/^(0x)?[0-9a-fA-F]{64}$/.test(userInput)) {
```

Comment:
The current private key format is an all-caps alphanumeric string and this checks for hex. We can support hex if that's a thing but we have to maintain support for the keys we create now.